app/views/content/assessment-only-providers.md

```diff
@@ -479,6 +479,11 @@ provider_groups:
       name: Andrew Locke
       telephone: 020 3194 3200
       email: institute@tesglobal.com
+inset_text:
+  international-content:
+    text: |-
+      If you’re a non-UK citizen, you should contact <a href="#non-uk">training providers who assess international teachers</a>. None of the other UK regional or national providers can help with non-UK enquiries. Please do not contact them if you are not in the UK.
```

Comment:
```suggestion
      If you’re a non-UK citizen, you should contact <a href="#group--non-uk">training providers who assess international teachers</a>. None of the other UK regional or national providers can help with non-UK enquiries. Please do not contact them if you are not in the UK.
```

@philspray Using a link of `non-uk` drops you to below the Non-UK title for some reason. I think if you use this group link, you will land above the Non-UK title which I think makes more sense as a user?